### DISCUSSION TITLE
What does shard mean?

### DISCUSSION THREAD

**Discussion by Tusenka** [prometheus/prometheus#15108] [General]
https://github.com/prometheus/prometheus/blob/a1c2f59bc577465e8b46b9cca6dca61ec7ecffe8/tsdb/db.go#L199

**Comment by bboreham** [comment]
The concept, as used in databases, is described here:  https://en.wikipedia.org/wiki/Shard_(database_architecture) and https://learn.microsoft.com/en-us/azure/architecture/patterns/sharding.

This member in the Prometheus code is to facilitate a feature in a downstream project described here: https://promcon.io/2022-munich/talks/anatomy-of-a-slow-query-and-less/
